fvt/predictor/predictor_suite_test.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this spaces for tab is go 1.19 linting result, as it thinks the copyright comment is the package documentation. I will do a sweep in the release prep PR where we update to go 1.19. For this PR, could you undo this change. If the linter complains, instead add an empty line between the copyright comment and the package declaration

Reply:
Thanks, I've undone the change (I had to add the empty line as you mentioned)